scripts/devtech/devtech_recipes.zs

```diff
@@ -77,4 +77,16 @@ for i in (id + 1) to id + 9 {
     tier += 1;
 }
 
-id += 9;
\ No newline at end of file
+id += 9;
+
+// Stock manipulators
+tier = 1;
+for i in (id + 1) to id + 9 {
+    recipes.addShaped("gt_stock_manipulator" + (i - id), machine.makeStack(i), [
+        [tieredGlass[tier], sensors[tier], <minecraft:clay_ball>],
```

Comment:
As much as I love the clay, this should change to tiered glass.

Reply:
:'(